axelrod/load_data_.py

```diff
@@ -1,8 +1,19 @@
-from typing import Dict, List, Tuple
+from typing import Dict, List, Text, Tuple
 
+import os
 import pkg_resources
 
 
+def axl_filename(axl_path: Text) -> Text:
```

Comment:
Could we add a (simple) test in a  `test_load_data.py` file to the tests. I realise this command is used throughout the tests but it could be helpful at a later stage to know that this function is working as expected in case something else breaks.

Reply:
Done.